frame/system/src/lib.rs

```diff
@@ -528,28 +521,16 @@ decl_module! {
 		/// The maximum number of blocks to allow in mortal eras.
 		const BlockHashCount: T::BlockNumber = T::BlockHashCount::get();
 
-		/// The maximum weight of a block.
-		const MaximumBlockWeight: Weight = T::MaximumBlockWeight::get();
```

Comment:
why not keeping weight info in const ?
```
		const BlockWeight: limits::BlockWeights = T::BlockWeights::get();
```
and derive codec for BlockWeights ?

Reply:
ah no I meant just put it so it is in metadata

Reply:
Oh, I didn't know that only these consts end up in the metadata. Let's bring it back then maybe?

Reply:
Yeah I guess better to bring back, though I don't know what usage external tools makes of it.